test/benchmarks/material/table/app.module.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {A11yModule} from '@angular/cdk/a11y';
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {BasicTable, GenericObject} from './basic-table';
+import {MatTableModule} from '@angular/material/table';
+
+/**
+ * @title Checkbox benchmark component.
+ */
```

Comment:
While addressing other feedback, you may want to remove this as it's not needed (and a leftover from docs examples)